Gemfile.lock

```diff
@@ -182,64 +182,62 @@ GEM
       addressable (~> 2.7)
     letter_opener (1.7.0)
       launchy (~> 2.2)
-    listen (3.4.1)
+    listen (3.5.1)
       rb-fsevent (~> 0.10, >= 0.10.3)
       rb-inotify (~> 0.9, >= 0.9.10)
-    loofah (2.9.0)
+    loofah (2.9.1)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
     mail (2.7.1)
       mini_mime (>= 0.1.1)
-    marcel (0.3.3)
-      mimemagic (~> 0.3.2)
+    marcel (1.0.1)
     method_source (1.0.0)
-    mimemagic (0.3.5)
-    mini_mime (1.0.2)
-    minitest (5.14.3)
+    mini_mime (1.0.3)
+    minitest (5.14.4)
     msgpack (1.4.2)
     net-scp (3.0.0)
       net-ssh (>= 2.6.5, < 7.0.0)
     net-ssh (6.1.0)
-    nio4r (2.5.5)
-    nokogiri (1.11.1-x86_64-darwin)
+    nio4r (2.5.7)
```

Comment:
### [nio4r](https://github.com/socketry/nio4r)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 2.5.5 → 2.5.7

[_[change-log](https://github.com/socketry/nio4r/blob/master/CHANGES.md), [source-code](https://github.com/socketry/nio4r/tree/v2.5.7)_]





<details>
<summary>Commits</summary>

A change of **8** commits. See the full changes on [the compare page](https://github.com/socketry/nio4r/compare/v2.5.5...v2.5.7).

These are the individual commits:
- (a7993fa) [CHANGES.md: add v2.5.5](https://github.com/socketry/nio4r/commit/a7993fa55197fc7676bab7d5d8af7fee99bbc992)
- (3c80de0) [CI: Remove the macos-11.0 target, for now](https://github.com/socketry/nio4r/commit/3c80de015adabcba584cdeef77aa94eb7a60dfa9)
- (cc6e0f7) [Avoid requiring linux kernel headers to be present](https://github.com/socketry/nio4r/commit/cc6e0f76c2fd51a341dec13f27160cc22b037878)
- (90d9599) [Suggest kqueue on OSX if >= 10.12.2](https://github.com/socketry/nio4r/commit/90d9599e5b7899f731bc8496dea46c2d42c3b61f)
- (9192212) [Patch version bump.](https://github.com/socketry/nio4r/commit/9192212525001c4d85228425cab83e1d102b617a)
- (3d1a49e) [Don't try to link universal extension.](https://github.com/socketry/nio4r/commit/3d1a49ef43e54fe549371ef32957a251d95234aa)
- (1242bf6) [Allow usage of native extension by default.](https://github.com/socketry/nio4r/commit/1242bf6a4eaf8456f8d7542cc4c4147c3fc16e62)
- (05532a5) [Patch version bump.](https://github.com/socketry/nio4r/commit/05532a5a35758c812498a1767c25118a7f64e101)


</details>